src/ontodia/diagram/elementLayer.tsx

```diff
@@ -107,24 +107,29 @@ class OverlayedElement extends React.Component<OverlayedElementProps, OverlayedE
         };
     }
 
-    private rerenderTemplate = () => this.setState({templateProps: this.templateProps()});
+    private rerenderTemplate = () => this.setState({ templateProps: this.templateProps() });
 
     render(): React.ReactElement<any> {
-        const {model, view, onResize, onRender} = this.props;
+        const { model, view, onResize, onRender } = this.props;
 
         this.typesObserver.observe(model.template.types);
         this.propertyObserver.observe(Object.keys(model.template.properties));
 
         const template = view.getElementTemplate(model.template.types);
 
-        const {x = 0, y = 0} = model.get('position') || {};
+        const { x = 0, y = 0 } = model.get('position') || {};
         let transform = `translate(${x}px,${y}px)`;
 
         const angle = model.get('angle') || 0;
         if (angle) { transform += `rotate(${angle}deg)`; }
+        
+        var divStyle;
+        if ( window.location.href == "http://localhost:10444/vowl.html" )
+             divStyle = {position: 'absolute', transform, borderStyle: "solid", borderWidth: '1px', borderRadius: '15px', borderColor: "grey", padding: '5px'}
```

Comment:
I think the better way to add this styles is to specify them in `.scss` for VOWL templates.

Reply:
I was asked not to clone this class for vowl. Therefore, it remains the same. Depending on the values in the select I override the necessary properties for ElementLayer.
I leave without changes.

Reply:
This module (`elementLayer`) shouldn't contain any domain-related logic, especially like overriding styles for specific URIs (it also wouldn't work on any other deployment URI, so it isn't a good idea anyway).

Instead, a right way to do this is to add this styles directly to VOWL-related templates' `scss`-styles, so they could work in any environment.